backend/fleet_management/api.py

```diff
@@ -76,6 +65,4 @@ class ProjectView(generics.ListAPIView):
     serializer_class = ProjectSerializer
 
     def get_queryset(self):
-        return Project.objects.filter(
-            country=self.request.user.country,
-        )
+        return Project.objects.filter(country=self.request.user.country,)
```

Comment:
it's also super strange, that it left this trailing `,` inside function arguments

Reply:
https://github.com/psf/black/issues/1288